pixi.toml

```diff
@@ -41,11 +41,11 @@ lint-py-fmt-check = "ruff format --check --config rerun_py/pyproject.toml"
 lint-py-blackdoc = "blackdoc --check"
 lint-py-mypy = "mypy --install-types --non-interactive --no-warn-unused-ignore"
 lint-py-ruff = "ruff check --config rerun_py/pyproject.toml"
-lint-taplo = "taplo fmt --check"
-lint-typos = "typos"
```

Comment:
Why remove `lint-typos`?
Unintentional?

Reply:
oops. yes. Thank you
I was thinking of removing it since it doesn't seem to be used on CI, but I didn't follow through on that (and don't intend to do now)